Makefile

```diff
@@ -16,6 +16,10 @@ ifndef ANTREA_PLATFORM
 	ANTREA_PLATFORM=kubernetes
 endif
 
+ifndef IS_CERTIFICATION
+	IS_CERTIFICATION=no
```

Comment:
I know with makefiles does not make a huge difference, but can we use true/false instead of yes/no? This mostly for consistency with the antrea makefile